src/__experimental__/components/radio-button/radio-button.style.js

```diff
@@ -17,6 +17,7 @@ const RadioButtonStyle = styled(CheckboxStyle)`
     theme,
     inline
   }) => css`
+    padding-top: 8px;
```

Comment:
How about adding the same `pt` prop defaulted to `1` (as in in `Checkbox` now) to `RadioButton` instead of adding this line?

Reply:
I could have, but only wanted to add styled-system to one component in this PR just as an example to people so I decided just to do this for now. If you really want me to change it I can? It will be getting updated soon anyway as I'll be going through all the input components and adding styled-system